schemas/json/dandi/upload.json

```
@@ -20,9 +20,15 @@
                     "min": 1,
                     "default": 1
                 },
+                "ignore_cache": {
+                    "type": "boolean",
+                    "description": "Ignore the cache and re-upload all files.",
```

```suggestion
                    "description": "Ignore the cache used by DANDI to speed up repeated operations.",
```

Keeping it more generic since I don't actually know every single way they interact with their cache. All I know is it supposedly speeds up certain things. Suspect it has to do mostly with expensive checksum calculations